src/lib/api/search.ts

```
@@ -0,0 +1,140 @@
+  symbol: string;
+  name: string;
+  exchange: string;
+  type: string;
+  isInWatchlist: boolean;
+};
+
+const FINNHUB_BASE_URL = (import.meta as any).env?.VITE_FINNHUB_BASE_URL as string | undefined;
+const FINNHUB_API_KEY = (import.meta as any).env?.VITE_FINNHUB_API_KEY as string | undefined;
+
+// Simple in-memory cache with TTL semantics per URL
+const cacheStore = new Map<string, { expiry: number; data: any }>();
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)